javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,27 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (class07Students.length == 6) {
+    // Checking the array lenght
+    console.log("Cannot add more students to class 07");
+  } // checking if the element already exists in array
+  else if (class07Students.includes(studentName)) {
+    console.log(studentName + "already exists");
+  } // if the student name is "queen", push
+  else if (studentName === "Queen") {
```

Comment:
I recommend to use `toLowerCase` method like this
```js
else if (studentName.toLowerCase() === "queen") {
```
then we can use `"queen"` or `"Queen"`

Reply:
Done!!